src/conf_mode/protocols_mpls.py

```diff
@@ -121,19 +121,31 @@ def apply(mpls):
         system_interfaces = []
         system_interfaces.append(((os.popen('sysctl net.mpls.conf').read()).split('\n')))
         del system_interfaces[0][-1]
-        for configured_interface in mpls['interface']:
-            for system_interface in system_interfaces[0]:
-                if configured_interface in system_interface:
-                    call(f'sysctl -wq net.mpls.conf.{configured_interface}.input=1')
-                elif system_interface.endswith(' = 1'):
-                    system_interface = system_interface.replace(' = 1', '=0')
-                    call(f'sysctl -wq {system_interface}')
+        system_interfaces = system_interfaces[0]
+        for system_interface in system_interfaces:
+            if system_interface.endswith(' = 1'):
+                replaced_once_system_interface = system_interface.replace('net.mpls.conf.', '')
+                replaced_twice_system_interface = replaced_once_system_interface.replace('.input = 1', '')
+                parsed_system_interface = replaced_twice_system_interface.replace('/', '.')
+                if parsed_system_interface in mpls['interface']:
```

Comment:
Please use a `not` condition instead of `pass` - makes it easier to read.

```
if parsed_system_interface not in mpls['interface']:
    call(f'sysctl -wq net.mpls.conf.{replaced_twice_system_interface}.input=0')
```

Reply:
Ahh, sure. I can make this change. Thank you :)